test_regress/t/t_tri_top_en_out_bad.pl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(vlt => 1);
+
+compile(
+  make_top_shell => 0,
+  make_main => 0,
+  verilator_make_gmake => 1,
+  verilator_flags2 => ["--exe --pins-inout-enables --no-timing -Wno-STMTDLY",
+                       "$Self->{t_dir}/$Self->{name}.cpp"]
```

Comment:
```suggestion
  verilator_flags2 => ["--exe --pins-inout-enables --no-timing -Wno-STMTDLY"]
```

Avoiding --exe means won't need to compile...